### ISSUE TITLE
Handle whitespaces when pulling scenarios by name

### ISSUE THREAD

**Issue by tikolakin** [hiptest/hiptest-publisher#70]
Hi
is there a chance to avoid explicitly encode spaces? So this work is to be done by hiptest-publisher
currently
The following succeed 
`./hiptest --token token --only tests --filter-on-folder-name="Basic%20Information"`

The following fails 
`./hiptest --token token --only tests --filter-on-folder-name="Basic Information"`

Thanks

**Comment by vincent-psarga** [comment]
Hi Tiko,

Thanks for the feedback, we'll try to fix that in the coming releases :)

**Comment by cbliard** [comment]
Hi Tiko,

I tried to reproduce your issue using hiptest-publisher 1.10

```
$ hiptest-publisher --token 12345 --filter-on-folder-name='Basic information' --verbose
Running Hiptest-publisher 1.10.0 with:
 - __cli_args: #<Set: {:token, :filter_on_folder_name, :verbose}>
 - __config_args: #<Set: {}>
[...]
Request sent to: https://hiptest.net/publication/12345/project?filter_folder_name=Basic%20information
```

It looks like the space character has been correctly encoded. Can you send us the output you have with `--verbose` option? (Remove your token from the output before posting)

**Comment by tikolakin** [comment]
thanks, we are running v1.7.1, that explains, thanks.

**Comment by cbliard** [comment]
You actually unveiled a bug. Some other characters are not properly escaped, like `&` or accentuated letters. We'll fix that in a future release.

**Comment by vincent-psarga** [comment]
Fix released in 1.10.1 :)
